### DISCUSSION TITLE
Change quota

### DISCUSSION THREAD

**Discussion by TicTac9k1** [ihexxa/quickshare#187] [General]
I've tested this and it does exactly what I want, but how/where do I changed the quota limit? 
I run the docker image.

**Comment by ihexxa** [comment]
@TicTac9k1 , sry for missing this thread.
It is a bit deep:
- you should login with the admin account and click settings in the right top.
- And go to "Management" -> "Users" -> click unfold icon (besides "delete").
- Then you can find it.
